test/modules/Modules.Test/TestResultCoordinator/Reports/DirectMethod/DirectMethodLongHaulReportGeneratorTest.cs

```diff
@@ -0,0 +1,394 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Modules.Test.TestResultCoordinator.Reports.DirectMethod
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Threading.Tasks;
+    using global::TestResultCoordinator.Reports;
+    using global::TestResultCoordinator.Reports.DirectMethod.LongHaul;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults;
+    using Microsoft.Azure.Devices.Edge.Storage;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Moq;
+    using Newtonsoft.Json;
+    using Xunit;
+
+    public class DirectMethodLongHaulReportGeneratorTest
```

Comment:
nit: I think it could increase readability to order the tests in the order of the DirectMethodLongHaulReportGenerator args.